app/presenters/publishing_api/role_presenter.rb

```diff
@@ -52,12 +52,24 @@ def polymorphic_path
       end
     end
 
+    def whip
+      return {} unless item.whip_organisation_id != nil
+
+      organisation = Whitehall::WhipOrganisation.find_by_id(item.whip_organisation_id) 
+
+      {
+        sort_order: organisation.sort_order,
+        label: organisation.label,
+      }
```

Comment:
Is the label here in another content item? If so, it would probably be better to use links, rather than the details of this content item, to avoid things getting out of sync.

Reply:
Renamed for clarity 👍